[2025-08-04T15:09:07.802+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-04T15:08:48+00:00 [queued]>
[2025-08-04T15:09:07.823+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-04T15:08:48+00:00 [queued]>
[2025-08-04T15:09:07.824+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-04T15:09:07.868+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): crea_vista_analitica> on 2025-08-04 15:08:48+00:00
[2025-08-04T15:09:07.872+0000] {standard_task_runner.py:60} INFO - Started process 88 to run task
[2025-08-04T15:09:07.876+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_medallon', 'crea_vista_analitica', 'manual__2025-08-04T15:08:48+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_medallon_dag.py', '--cfg-path', '/tmp/tmpt180blq_']
[2025-08-04T15:09:07.878+0000] {standard_task_runner.py:88} INFO - Job 165: Subtask crea_vista_analitica
[2025-08-04T15:09:07.992+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-04T15:08:48+00:00 [running]> on host 70eaa744a05c
[2025-08-04T15:09:08.162+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo.biasotto' AIRFLOW_CTX_DAG_ID='pipeline_medallon' AIRFLOW_CTX_TASK_ID='crea_vista_analitica' AIRFLOW_CTX_EXECUTION_DATE='2025-08-04T15:08:48+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-04T15:08:48+00:00'
[2025-08-04T15:09:08.164+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-04T15:09:08.165+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/scripts/postgres_04_crea_vista_analitica.py']
[2025-08-04T15:09:08.172+0000] {subprocess.py:86} INFO - Output:
[2025-08-04T15:09:08.354+0000] {subprocess.py:93} INFO - Conectado a PostgreSQL en postgres:5432/medallon_db
[2025-08-04T15:09:08.355+0000] {subprocess.py:93} INFO - Ejecutando: /opt/airflow/sql/04_vista_analisis.sql
[2025-08-04T15:09:08.355+0000] {subprocess.py:93} INFO - Vista de an√°lisis creada/reemplazada correctamente
[2025-08-04T15:09:08.365+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-04T15:09:08.416+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=pipeline_medallon, task_id=crea_vista_analitica, execution_date=20250804T150848, start_date=20250804T150907, end_date=20250804T150908
[2025-08-04T15:09:08.492+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-04T15:09:08.538+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
